src/it/MPMD-266-typeresolution-aggregated/invoker.properties

```diff
@@ -15,6 +15,6 @@
 # specific language governing permissions and limitations
 # under the License.
 
-invoker.goals = clean verify
+invoker.goals = clean install
```

Comment:
I really hope this change it required. `clean verify` should do its job, otherwise we have another issue.

Reply:
with only verify there was a complain about artifact not found.